src/configs/index.js

```diff
@@ -454,3 +454,10 @@ export const deliveryTypes = {
 };
 
 export const CY_CODE_ERR = 'code-error';
+
+export const SESSION_STORAGE = {
+  CHECKOUT_FORM: 'checkoutForm',
+  DELIVERY_TYPE: 'deliveryType'
+};
+
+export const HORONDI = 'horondi';
```

Comment:
you can use this const in  **src/index.js**  in localStorage/sesstionStorage conditionals